fontbe/src/glyphs.rs

```diff
@@ -110,46 +116,119 @@ fn create_composite(
     fn exec(&self, context: &Context) -> Result<(), Error> {
         trace!("BE glyph work for {}", self.glyph_name);
 
         let static_metadata = context.ir.get_final_static_metadata();
         let var_model = &static_metadata.variation_model;
-        let default_location = var_model.default_location();
+        let default_location = static_metadata.default_location();
         let ir_glyph = &*context.ir.get_glyph_ir(&self.glyph_name);
         let glyph: CheckedGlyph = ir_glyph.try_into()?;
 
         // Hopefully in time https://github.com/harfbuzz/boring-expansion-spec means we can drop this
         let glyph = cubics_to_quadratics(glyph);
 
-        // TODO refine (submodel) var model if glyph locations is a subset of var model locations
-
-        match glyph {
+        let (name, point_seqs) = match glyph {
             CheckedGlyph::Composite { name, components } => {
                 let composite = create_composite(context, &name, default_location, &components)?;
-                context.set_glyph(name, composite.into());
+                context.set_glyph(name.clone(), composite.into());
+                (name, point_seqs_for_composite_glyph(ir_glyph))
             }
-            CheckedGlyph::Contour { name, contours } => {
-                // Draw the default outline of our simple glyph
-                let Some(path) = contours.get(default_location) else {
+            CheckedGlyph::Contour { name, paths } => {
+                // Convert paths to SimpleGlyph so we can get consistent point streams
+                let instances = paths
+                    .into_iter()
+                    .map(|(loc, path)| {
+                        match SimpleGlyph::from_kurbo(&path).map_err(|e| Error::KurboError {
+                            glyph_name: self.glyph_name.clone(),
+                            kurbo_problem: e,
+                            context: path.to_svg(),
+                        }) {
+                            Ok(glyph) => Ok((loc, glyph)),
+                            Err(e) => Err(e),
+                        }
+                    })
+                    .collect::<Result<HashMap<_, _>, Error>>()?;
+
+                // Establish the default outline of our simple glyph
+                let Some(base_glyph) = instances.get(default_location) else {
                     return Err(Error::GlyphError(ir_glyph.name.clone(), GlyphProblem::MissingDefault));
                 };
-                let base_glyph = SimpleGlyph::from_kurbo(path).map_err(|e| Error::KurboError {
-                    glyph_name: self.glyph_name.clone(),
-                    kurbo_problem: e,
-                    context: path.to_svg(),
-                })?;
-                context.set_glyph(name, base_glyph.into());
+                context.set_glyph(name.clone(), base_glyph.clone().into());
+
+                (name, point_seqs_for_simple_glyph(ir_glyph, instances))
             }
-        }
+        };
+
+        // Everybody loves gvar!
```

Comment:
I worry this comment may be misleading to people who are unfamiliar with gvar

Reply:
Ha, that is a bit cryptic. Updated to more clearly indicate gvar applies to both types of glyph.